api/db-cdr/generate-cdr/cb-criteria-refactor/debug/comp-tables.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+BQ_PROJECT='all-of-us-ehr-dev'       # project
```

Comment:
Since this is being committed, these three variables should probably be passed as parameters to the script rather than being hard coded.